CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## unreleased
 * Remove use of deprecated `setNetworkActivityIndicatorVisible` on iOS 13+ (the network activity indicator was removed from the status bar in iOS 13) (fixes #379)
+* Require `braintree_ios` 5.12.0 or higher
+  * Adds support for iOS 16 and Xcode 14
```

Comment:
Take it or leave it, but should we move this point for adding support for iOS 16 + Xcode 14 to the main level vs nested? Wondering if this reads like we are just adding that support for core but not the drop-in?

Reply:
Good idea! Fixed